src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -217,17 +198,21 @@ The network throughput of the load generator machine, as well as the SUT will li
 
 k6 needs a significant amount of memory when uploading files, as every VU is independent and has its own memory.
 
-### Data transfer costs
+### Costs
+
+#### Data transfer
```

Comment:
Getting into `h4s` is kind of "prose smell". So much nesting makes it hard for the reader to understand what this relates to.

In any case, we can at least use description lists: https://github.com/grafana/k6-docs/blob/main/CONTRIBUTING/gatsby-reference.md#description-list

Elsewhere, it probably makes sense to flatten the structure. I'll look into this.

Reply:
> saw some examples of how description lists are rendered, and I'm not sure if they would be a good fit here. > The titles are too large, and we don't need the indentation of the bodies.

I agree, I wont to reduce the size of description list items, but that's a CSS change. Anyway, they're pretty much the size of the h4s so no difference there.

The indentation is fine, IMO: it creates a clear sequence of repetition and alignment. With h4 there's only one level of alignment, the definition makes a clearer sequence of Item and description.

It would look like this: https://k6.io/blog/reverse-load-test/#analysis-bottlenecks-and-planned-improvements

Really I think it's fine.

Anyway the docs table of contents shows only h2s, so the best way to improve that navigation is by flattening the structure out, reducing h3s and h4s.